### PR TITLE
Upgrade to hyrax 2.8.0

### LINKED ISSUE #17: Bump hyrax version to 2.8.0

### PR DESCRIPTION
**Pull request by crowesn**
Fixes #17 

Bump hyrax version to 2.8.0 and upgrade dependencies. Started from scratch by generating new hyrax-based app, running generators, configuring rubocop, etc.

Changes proposed in this pull request:
* Hyrax 2.8.0
* ruby version 2.6.3 -> 2.7.1
* rails version 5.1.6 -> 5.2.4

### CODE REVIEW

**Comment by hortongn** [comment]
@crowesn Thanks!  Will review today.

**Comment by crowesn** [comment]
https://github.com/samvera/hyrax/issues/4393

File: config/environments/development.rb [hunk 1]
```diff
@@ -39,6 +43,9 @@
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
+  # Highlight code that triggered database queries in logs.
+  config.active_record.verbose_query_logs = true
+
```
Comment (hortongn) [inline config/environments/development.rb:48]: Background jobs are not running at all.  Please add `config.active_job.queue_adapter = :inline`

Comment (crowesn) [reply config/environments/development.rb:48]: Got em, thanks.

**Pull request merged**